apps/lib/http_server.c

```diff
@@ -178,16 +178,16 @@ void spawn_loop(const char *prog)
                 }
             }
             if (i >= n_responders) {
-                log_HTTP(prog, LOG_ERR,
-                         "fatal: internal error: no free child slots");
+                log_HTTP(prog, LOG_CRIT,
+                         "internal error: no free child slots");
                 killall(1, kidpids);
             }
             break;
         }
     }
 
     /* The loop above can only break on termsig */
-    log_HTTP1(prog, LOG_INFO, "terminating on signal: %d", termsig);
+    log_HTTP1(prog, LOG_INFO, "Terminating on signal: %d", termsig);
```

Comment:
I do not understand the logic why you sometimes change the initial from lc to uc and sometimes the other way around.

Reply:
Oops, I had overlooked your review comment for a while

> I do not understand the logic why you sometimes change the initial from lc to uc and sometimes the other way around.

Well, I had to take a closer look again myself and then recalled that the idea was to use lower-case for errors (including fatal ones) and upper case for all other (intermediate) log output such as warnings, info, and traces.
Doing so, I had overlooked two cases, which I've just made consistent with this style.

Reply:
This does not make much sense to me. Why would you distinguish errors from _something else_ by the initial case?

Reply:
Good point .
The sort of message is already indicated directly by other means.

So converted all log messages to start in lower case (except of course where an upper-case acronym such as `RAND_poll()` is in front).